eos-updater-flatpak-installer/tests/Makefile.am

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2017 Endless Mobile, Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+include $(top_srcdir)/glib-tap.mk
+
+installed_testdir = $(libexecdir)/installed-tests/libeos-updater-flatpak-installer-@EUU_API_VERSION@
+installed_test_metadir = $(datadir)/installed-tests/libeos-updater-flatpak-installer-@EUU_API_VERSION@
+
+# Flags for all test binaries
+AM_CPPFLAGS = \
+	-I$(abs_top_srcdir) \
+	-I$(abs_top_builddir) \
+	-include "config.h" \
+	-DG_LOG_DOMAIN=\"libeos-updater-flatpak-installer-tests\" \
+	$(NULL)
+AM_CFLAGS = \
+	$(STD_CFLAGS) \
+	$(CODE_COVERAGE_CFLAGS) \
```

Comment:
Drop `$(CODE_COVERAGE_CFLAGS)` here, since we don’t want to measure code coverage of the tests themselves.